prisma-fmt/tests/code_actions/scenarios/relation_mode_prisma_missing_index/diagnostics.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "range": {
+            "start": {
+                "line": 19,
+                "character": 20
+            },
+            "end": {
+                "line": 19,
+                "character": 63
+            }
+        },
+        "severity": 1,
+        "message": "With `relationMode = \"prisma\"`, no foreign keys are used, so relation fields will not benefit from the index usually created by the relational database under the hood. This can lead to poor performance when querying these fields. We recommend adding an index manually. Learn more at https: //pris.ly/d/relation-mode#indexes"
```

Comment:
The space after "https:" is surprising. Is that just a "visual" artifact here in the test or a bug?

Reply:
Huh, it's actually there in the diagnostic, not sure where that came from 😅 (this isn't auto-generated by anything currently, I manually made this so it doesn't reflect anything anywhere else)

Reply:
Ah, so this does not reflect actual funcitonality but logic and expectation of a test at the same time?

Reply:
As far as I've understood it, _yes_